src/Avalonia.Controls/MenuItem.cs

```diff
@@ -337,6 +338,9 @@ protected override IItemContainerGenerator CreateItemContainerGenerator()
 
         protected override void OnAttachedToLogicalTree(LogicalTreeAttachmentEventArgs e)
         {
+            if (_hotkey != null)
+                // Control attached again, set Hotkey to create a hotkey manager for this control
```

Comment:
please use braces, even though its a single statement...

and put the comment next to the if () or above it.

Reply:
Added, thanks